website/content/gateway/api_reference/resources/authorizations/authorizations_methods/token.md

```diff
@@ -10,3 +10,93 @@ Two token frameworks are supported:
+{{% description_details %}}Token Account Number (TAN) to charge.
+
+Found in `encryptedPayload.encryptedData.accountNumber` (MDES) or `encryptedPayload.token.paymentToken` (SCOF).
+{{% /description_details %}}
+
+{{% description_term %}}token[m4m][expire_month] {{% regex %}}[0-9]{2}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Expiry month of token to charge.
+
+Found in `encryptedPayload.encryptedData.applicationExpiryDate` (MDES) or `encryptedPayload.token.tokenExpirationMonth` (SCOF).
+{{% /description_details %}}
+
+{{% description_term %}}token[m4m][expire_year] {{% regex %}}20[0-9]{2}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Expiry year of token to charge.
+
+Found in `encryptedPayload.encryptedData.applicationExpiryDate` (MDES) or `encryptedPayload.token.tokenExpirationYear` (SCOF).
+{{% /description_details %}}
+
+{{% description_term %}}token[m4m][eci] {{% regex %}}0[267]{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Electronic Commerce Indicator.
+
+Found in `eci` (SCOF).
+{{% /description_details %}}
+
+{{% description_term %}}token[m4m][cryptogram] {{% regex %}}[:base64:]{28}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Token cryptogram.
+
+Must start with `[A-P]` to be a Digital Secure Remote Payments (DSRP) cryptogram.
+
+Found in `encryptedPayload.encryptedData.de48se43Data` (MDES) or `encryptedPayload.dynamicData.dynamicDataValue` (SCOF).
+{{% regex_optional %}}Required for cardholder-initiated transactions; otherwise optional.{{% /regex_optional %}}
+{{% /description_details %}}
+
+{{% description_term %}}token[m4m][3dsecure][v2] {{% regex %}}dictionary{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}See [Authentication: [3dsecure][v2]](#authentication-3dsecure-v2).
+{{% regex_optional %}}Optional{{% /regex_optional %}}
+{{% /description_details %}}
+{{% /description_list %}}
+
+#### Method: token[vts]
+
+The required values are found in the VTS provision token response.
+
+{{% description_list %}}
+{{% description_term %}}token[vts][tan] {{% regex %}}[0-9]{12,19}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Token Account Number (TAN) to charge.
+
+Found in `tokenInfo.encTokenInfo`.
+{{% /description_details %}}
+
+{{% description_term %}}token[vts][expire_month] {{% regex %}}[0-9]{2}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Expiry month of token to charge.
+
+Found in `tokenInfo.expirationDate.month`.
+{{% /description_details %}}
+
+{{% description_term %}}token[vts][expire_year] {{% regex %}}20[0-9]{2}{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Expiry year of token to charge.
+
+Found in `tokenInfo.expirationDate.year`.
+{{% /description_details %}}
+
+{{% description_term %}}token[vts][eci] {{% regex %}}0[57]{{% /regex %}}{{% /description_term %}}
+{{% description_details %}}Electronic Commerce Indicator.
+
+Found in `cryptogramInfo.eci`.
```

Comment:
Similar to https://github.com/clearhaus/gateway-api-docs/pull/149/files#r983326754

We need optional due to subsequent recurring.

Reply:
I've pushed https://github.com/clearhaus/gateway-api-docs/pull/149/commits/93fc4b17d368fe72278163dc91d847fe364e4eb5. I'll resolve this thread. We can start a new one.